### PR TITLE
Consolidates mission statement

### LINKED ISSUE #867: Consolidate "mission statement" documentation
Need to consolidate the following:

* [GH wiki](https://github.com/AgileVentures/WebsiteOne/wiki/Project-Mission%2C-Goals-and-Vision)
* [Google Doc](https://docs.google.com/document/d/1KM3QF_iyAd7jkxiSd2hQOWl1vSzwuJbd0inKF6JziTU/edit)

### PR DESCRIPTION
**Pull request by tansaku**
fixes #867 

When this gets merged we should ensure that the relevant pages get deleted in the:

* [ ] Github Wiki
* [ ] AV Wiki
* [ ] AV Pitch text

### CODE REVIEW

File: docs/mission_statement.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## [WebsiteOne](http://agileventures.org/)
+## Mission statement
+#### To provide a platform for online collaboration and crowdsourced project development.
```
Comment (tansaku) [inline docs/mission_statement.md]: I'm thinking to replace with "To provide a coordination point for new members hoping to join existing teams, start new projects and connect with charitable causes"

**Comment by tansaku** [comment]
I'm thinking to merge the following text from [this google doc](https://docs.google.com/document/d/1KM3QF_iyAd7jkxiSd2hQOWl1vSzwuJbd0inKF6JziTU/edit) into this document:

> The AgileVentures website provides a coordination point for new members hoping to join existing teams, start new projects and connect with charitable causes.  Existing teams generally have autonomy in how they run their projects but benefit from guidance, support and input from senior Agile Ventures members.

> Each project is autonomous in terms of choosing technologies and workflow, although Agile Ventures promotes a short cycle development process with prototypes being delivered to clients at regular intervals (e.g. weekly) for feedback that allows a team to stay on track to best support customer needs.

> We are deeply committed to open source (all code shared publicly) and open development (open meetings and documentation), and we expect all teams to be similarly committed.  We also recommend that each project maintains its documentation to be as terse and as up to date as possible to facilitate the on-boarding of new members.  The nature of volunteer open source projects means a high turnover of participating members; ideally it will be as straightforward as possible to contribute value to a project.

The AgileVentures web site (WSO) has been a learning project for many developers over the last few years, and huge amounts of work have been put in to get it where it currently is.  It is a testament to the dedication and commitment of everyone who has been involved.  However as a consequence of it being a learning project where a number of visions have been pursued it unfortunately lacks coherency.

> Moving forward WSO needs more coherent design; a new phase for the website where the focus is on doing few things (or perhaps one) exceptionally well rather than being a place for many novice developers to implement whatever features they are interested in. There are plenty of opportunities for learning in the increasing number of new projects. The vision for the website is to support the AV projects and edX MOOC pair programming activities as effectively as possible in line with the values outlined above.  In order to move forward coherently the final judgements on the site UX, codebase and workflow will be made by the senior project manager.   This is because WSO does not have a clear non-technical customer who can define the single vision like some other AV projects.

File: docs/expanded_mission_statement.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+The AgileVentures web site (WSO) has been a learning project for many developers over
+the last few years, and huge amounts of work have been put in to get it where it
+currently is.  It is a testament to the dedication and commitment of everyone who has
+been involved.  However it lacks coherency since it was an experimental learning
+project where a number of competing visions have been pursued.
+
+Moving forward WSO will have a more coherent design focus; a new phase for the website
+where the emphasis is on doing few things (or perhaps even just one) exceptionally well
+rather than being an entry level project where developers implement whichever
```
Comment (tansaku) [inline docs/expanded_mission_statement.md]: in response to comments from @sidhene and @freeranger I changed this to "entry-level project" and away from a project for "learning" or "novice" developers.

**Comment by tochman** [comment]
I would also suggest that you update the pitch text on WSO project page.

**Comment by diraulo** [comment]
If there's no objections I'll go ahead and merge this.

**Comment by tansaku** [comment]
@diraulo when you merge ping me so I can make the other changes in the other wikis etc.

**Comment by diraulo** [comment]
done @tansaku

**Pull request merged**